.github/workflows/delete-translation.yml

```
@@ -2,22 +2,22 @@ name: Crowdin Branch Clean-Up
 
 on:
   pull_request:
-    branches:
-      - 'translation-*'
-      - 'stp-*'
     types: [closed]
+    branches:
+        - 'master*'
 
 jobs:
   delete_translation_branch:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
       - name: Get branch name
         id: branch-name
         uses: tj-actions/branch-names@v5.2
-      - name: Fetching Feature Branch
-        run: |
-          echo "Latest branch merged to master: ${{ steps.branch-name.outputs.head_ref_branch }}"
+      - name: Checking branch name (❌ Auto stop on non-translation branches)
+        run: exit 1
```

```suggestion
        run: exit 0
```
This will consider the workflow as success